src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/DbContextWithNewTransactionFactory.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Data.Common;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+
+namespace SFA.DAS.CommitmentsV2.Data
+{
+    public class DbContextWithNewTransactionFactory : IDbContextFactory
```

Comment:
The name of the class suggests that this is a factory for creating DbContexts that will have a new transaction but I don't see anything in this class which would result in a new transaction being created?

Reply:
Agreed, changed to DbContextFactory